src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,51 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static removeByName(animalName) {
+    Animal.alive = Animal.alive.filter(x => x.name !== animalName);
+  }
+
+  static checkHealth(herbivore) {
+    if (herbivore.health === 0) {
+      Animal.removeByName(herbivore.name);
+    }
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+
+  constructor(name, health = 100, hidden = false) {
```

Comment:
```suggestion
  constructor(name) {
```
it should take only name to create an instance of a class